src/bindings/glib/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ else()
 	target_compile_options (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_CFLAGS_OTHER})
 
 	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC elektra-core elektra-kdb)
-	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LDFLAGS})
+	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LIBRARIES})
```

Comment:
erm yeah. Forget that.
You can change `PUBLIC ${GOBJECT_LDFLAGS}` to `PRIVATE ${GOBJECT_LDFLAGS}` and additionally add $GOBJECT_LIBRARIES. But that is actually a misuse of this feature and a possible subproject linking to gelektra needs to lookup and add $GOBJECT_LDFLAGS on it's own.